### PR TITLE
Added mysql support to setting

### PR DESCRIPTION
**Pull request by Lastwisher**
I added some mysql support to setting and added a settings adapter so in future if we need we can add more supports for settings etc.. 

Also i changed curl cookies to store at database, i will make full documentation if you like to pull.

Thanks.

### CODE REVIEW

File: src/Instagram.php [hunk 1]
```diff
@@ -15,37 +15,47 @@ class Instagram
...
     public $isLoggedIn = false; // Session status
     public $rank_token;         // Rank token
-    public $IGDataPath;         // Data storage path
-    public $customPath = false;
+
+
     public $http;
     public $settings;
+
+    public $settingsAdopter         = ["type" => "file",
```
Comment (IndraGunawan) [inline src/Instagram.php]: why the variable name `$settingAdopter` not `$settingAdapter`?

Comment (Lastwisher) [inline src/Instagram.php]: typo*

**Comment by mgp25** [comment]
@Lastwisher can you fix typos and try to send PR with the style fixes? + Can you post here .db structure?

**Comment by Lastwisher** [comment]
Hi mgp
Ill make few commits, first i upgrade code a bit, I will add structure, which format do you like ? 

Im also working on response and request system now, im mapping all response objects by json mapper, and adding missing fields on response objects. Ill create a PR for that when we done with this one. Its will clear code alot and bring some decent format for all of responses. And code will allow very easy update for future.

I thing i will send commit for settings integration today, can you send me your skype for easier comminication. 

Thanks.

**Comment by mgp25** [comment]
@Lastwisher send me an email (check my profile)

**Comment by Lastwisher** [comment]
Sended.

**Pull request merged**